src/api/useAuth.jsx

```diff
@@ -8,29 +8,37 @@ import { addUserToDatabase } from './firebase.js';
  * the button redirects the user to the Google OAuth sign-in page.
  * After the user signs in, they are redirected back to the app.
  */
-export const SignInButton = () => (
-	<button
-		type="button"
-		onClick={() => signInWithRedirect(auth, new GoogleAuthProvider())}
-	>
-		Sign In
-	</button>
-);
+export const SignInButton = () => {
+	try {
+		signInWithRedirect(auth, new GoogleAuthProvider());
+	} catch (error) {
+		console.log('error', error);
+	}
+};
 
 /**
  * A button that signs the user out of the app using Firebase Auth.
  */
-export const SignOutButton = () => (
-	<button
-		type="button"
-		onClick={() => {
-			auth.signOut();
-			localStorage.clear();
-		}}
-	>
-		Sign Out
-	</button>
-);
+export const SignOutButton = () => {
+	try {
+		auth.signOut();
+	} catch (error) {
+		console.log('error', error);
+	}
+};
+
+// <button
```

Comment:
I like the changes in this file. Maybe we can rename the functions as they are not returning a button anymore.
Also I think comments can be deleted.

Reply:
I will change that in my next commit.